ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.ts

```diff
@@ -8,5 +9,83 @@ import {Component, OnInit} from '@angular/core';
 export class MainScreenComponent implements OnInit {
 	constructor() {}
 
-	ngOnInit() {}
+	tripFormData: any; // TODO
+	isHiddenTripRegistration: boolean;
+	userState: string;
+	isSavedRoute: boolean;
```

Comment:
instead of creating hundreds of isSown create trip state field. For example:
New = 0 // absolutely new trip
Saved = 1 // 
Submitted = 2

Reply:
Will be fixed in the next PR